compose/sdoc-server.yml

```diff
@@ -0,0 +1,27 @@
+services:
+  sdoc-server:
+    image: seafileltd/sdoc-server:0.8.0
+    restart: unless-stopped
+    container_name: sdoc-server
+    environment:
+      - DB_HOST=mariadb
+      - DB_PORT=3306
+      - DB_USER=${DB_USER:-root}
+      - DB_PASSWD=${SEAFILE_MYSQL_ROOT_PASSWORD:?Variable is not set or empty}
+      - DB_NAME=sdoc_db
+      - TIME_ZONE=${TIME_ZONE}
+      - SDOC_SERVER_LETSENCRYPT=false
+      - SDOC_SERVER_HOSTNAME=https://${SEAFILE_SERVER_HOSTNAME}/sdoc-server
```

Comment:
Afaict `SDOC_SERVER_HOSTNAME` is only used when generating the NGINX configuration, which is irrelevant since the NGINX inside `seafile-server` directly communicates with the services inside this container (on port `7070`, plus seahub <=> `seadoc_convert` on port `8888`).